docs/management/api/kill-process-api.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[kill-process-api]]
 === Terminate a process
 
-Terminates a process on a host running {elastic-defend}.
+Terminates a process on a host running {elastic-defend} or a supported agent type.
```

Comment:
We've been using "third-party" to describe response actions, so this edit is consistent with that:
```suggestion
Terminates a process on a host running {elastic-defend} or a supported third-party agent type.
```